src/components/explorer/ExplorerSearchResultsTable.js

```diff
@@ -29,11 +29,25 @@ const ExplorerSearchResultsTable = ({
 }) => {
     const { dataset } = useParams();
     const [currentPage, setCurrentPage] = useState(initialCurrentPage || 1);
-    const [numResults] = useState(data.length);
+    const [activeFilters, setActiveFilters] = useState([]);
 
     const [summaryModalVisible, setSummaryModalVisible] = useState(false);
     const [tracksModalVisible, setTracksModalVisible] = useState(false);
 
+    const filteredData = useMemo(() => {
+        return data.filter((item) => {
+            return Object.keys(activeFilters).every((filterKey) => {
+                const filterValues = activeFilters[filterKey] || [];
+                if (filterValues.length === 0) {
+                    return true;
+                }
+                return filterValues.includes(item[filterKey]);
+            });
+        });
+    }, [data, activeFilters]);
+
+    const numResults = useMemo(() => filteredData.length, [filteredData]);
```

Comment:
this memo doesn't do much, since `.length` is a prop of filteredData that doesn't take much time to compute and isn't an object. I'd remove this.